src/schema/schema_create.c

```diff
@@ -70,24 +77,25 @@ __check_imported_ts(WT_SESSION_IMPL *session, const char *uri, const char *confi
 
     /* Now iterate over each checkpoint and compare the aggregate timestamps with our oldest. */
     WT_CKPT_FOREACH (ckptbase, ckpt) {
-        if (ckpt->ta.newest_start_durable_ts > txn_global->oldest_timestamp)
+        if (ckpt->ta.newest_start_durable_ts > ts)
             WT_ERR_MSG(session, EINVAL,
               "%s: import found aggregated newest start durable timestamp newer than the current "
-              "oldest timestamp, newest_start_durable_ts=%" PRIu64 ", oldest_ts=%" PRIu64,
-              uri, ckpt->ta.newest_start_durable_ts, txn_global->oldest_timestamp);
+              "%s timestamp, newest_start_durable_ts=%" PRIu64 ", %s_ts=%" PRIu64,
+              uri, ts_name, ckpt->ta.newest_start_durable_ts, ts_name,
+              txn_global->oldest_timestamp);
```

Comment:
It should be `ts`.

Reply:
oh, my bad. Curious why the complier doesn't catch this.